### DISCUSSION TITLE
Added annotations

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Quantum-Codes/ScraGet#16] [Announcements]
*Annotations* also called *type hints* are added! Suggested by @ninjamar!
Annotations will help you determine the input type of the function arguement!

Also, Install requirements were defined so if you don't have the packages that makes ScraGet work (only requests module lol), it will automatically be installed!

The printing of this repository's link is removed.

<hr /><em>This discussion was created from the release <a href='https://github.com/Quantum-Codes/ScraGet/releases/tag/v0.1.8'>Added annotations</a>.</em>

**Comment by ninjamar** [comment]
Yay
